CBinding/CMake/CMakeFileFormat.cs

```diff
@@ -163,6 +168,8 @@ void ParseCommand (Match commandMatch)
 			case "add_library":
 			case "add_executable":
 				targetCommands.Add (command);
+				tempPath = new FilePath(string.Format ("{0}{1}", "./", command.Arguments [0].ToString ()));
```

Comment:
1. There's an implicit conversion from string to FilePath
2. String.Format calls ToString automatically on non-string arguments
3. Consider using [interpolated strings](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/interpolated-strings) instead of string.Format, they're more readable
4. You don't need to parameterize the string constant